JitsiConferenceEventManager.js

```diff
@@ -413,13 +413,13 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
 
     chatRoom.addListener(XMPPEvents.JSON_MESSAGE_RECEIVED,
         (from, payload) => {
-            const id = Strophe.getResourceFromJid(from);
-            const participant = conference.getParticipantById(id);
+            const _id = Strophe.getResourceFromJid(from);
+            const participant = conference.getParticipantById(_id);
 
-            if (participant) {
+            if (participant || _id === 'focus') {
```

Comment:
Do we have a better way to detect this?

Reply:
This is now gone. The new NON_PARTICIPANT_MESSAGE_RECEIVED event is used instead.